### PR TITLE
Vote proposal CLI cmd

### PR DESCRIPTION
**Pull request by mpetrun5**
Implement vote proposal cli cmd. Added simulate before voting to improve error message when voting.

## Related Issue Or Context

Closes: #140 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [x] I have updated the documentation locally and in chainbridge-docs.
- [x] I have added tests to cover my changes.
- [x] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: chains/evm/cli/bridge/vote-proposal.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	"github.com/ChainSafe/chainbridge-core/util"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+var voteProposalCmd = &cobra.Command{
+	Use:   "vote-proposal",
+	Short: "Vote on proposal",
+	Long:  "Votes on an on-chain proposal. Valid relayer private key required for transaction to be successful.",
```
Comment (MakMuftic) [inline chains/evm/cli/bridge/vote-proposal.go:22]: This is not following our new CLI naming guideline (only place where you can see it, for now, is [this closed PR](https://github.com/ChainSafe/chainbridge-core/pull/196)), but I guess this is something we should add somewhere - maybe even inside README file? @waymobetta @P1sar what do you think?

**Review by MakMuftic — CHANGES_REQUESTED** [review]

File: chains/evm/cli/bridge/vote-proposal.go (hunk 1 above)
Comment (MakMuftic) [reply chains/evm/cli/bridge/vote-proposal.go:22]: To be more precise, only this `Long` description is not following guidelines.

Comment (mpetrun5) [reply chains/evm/cli/bridge/vote-proposal.go:22]: Yeah, will update. Wrote this before the guideline.

**Comment by MakMuftic** [comment]
As I think @mpetrun5 approach on testing this CLI command was great, I created a separate issue for adding these kinds of unit tests for all our CLI commands. This would test our validation/processing/binding functions for flags and also the execution function for each CLI command.

**Review by P1sar — APPROVED** [review]

**Review by nmlinaric — APPROVED** [review]

**Review by MakMuftic — APPROVED** [review]

**Pull request merged**